### PR TITLE
Separate chain calls

### PR DESCRIPTION
**Pull request by sankaest**
| Q             | A
| ------------- | ---
| Readability fix  | according to https://github.com/yiisoft/docs/issues/155, chain calls must be put on separate lines

### CODE REVIEW

File: tests/AssetBundleTest.php [hunk 1]
```diff
@@ -113,7 +113,11 @@ public function testBasePathEmptyWithAssetManagerWithBasePath(): void
 
     public function testBasePathEmptyBaseUrlEmptyWithAssetManagerWithBasePathWithBaseUrl(): void
     {
-        $manager = $this->manager->withLoader($this->loader->withBasePath('@asset')->withBaseUrl('@assetUrl'));
+        $manager = $this->manager->withLoader(
+            $this->loader
+            ->withBasePath('@asset')
```
Comment (terabytesoftw) [inline tests/AssetBundleTest.php:118]: shouldn't it be bleeding?

```php
        $manager = $this->manager->withLoader(
            $this->loader
                ->withBasePath('@asset')
```

Comment (sankaest) [reply tests/AssetBundleTest.php:118]: Sry, I am new here :) . 
What do You mean with bleeding? Can you correct it here, so I can update and use this style in future corrections?

Comment (sankaest) [reply tests/AssetBundleTest.php]: Found. You are right. My mistake. Will make new PR.

Comment (terabytesoftw) [reply tests/AssetBundleTest.php]: Thanks for the help 👍

**Review by terabytesoftw — APPROVED** [review]

**Review by vjik — APPROVED** [review]

**Review by devanych — APPROVED** [review]

**Pull request merged**